package.json

```diff
@@ -14,7 +14,9 @@
     "dev:auth": "cd microservices/auth && npm run dev",
     "dev:user-service": "cd microservices/user-service && npm run start",
     "dev:questions-service": "cd microservices/questions-service && npm run dev",
-    "dev-all": "concurrently -c \"auto\" \"npm:dev:*\""
+    "dev:rabbitMQ": "cd microservices/matching-service && npm run start:rabbitMQ",
+    "dev:matching-service": "cd microservices/matching-service && npm run dev",
```

Comment:
Do we need an install script for the `matching-service` too?

Reply:
Fixed in [efc7183](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/21/commits/efc71839b5b087bd24a8e1f3e04f623501b7be4b)